src/distilabel/llms/huggingface/inference_endpoints.py

```diff
@@ -74,13 +74,13 @@ class InferenceEndpointsLLM(AsyncLLM, MagpieChatTemplateMixin):
         `:hugging:`
 
     Examples:
-        Free serverless Inference API:
+        Free serverless Inference API (set the `input_batch_size=1` to avoid overloadings!):
```

Comment:
I'd be more specific:

Set the input_batch_size of the Task that uses this to avoid Model is overloaded.

Important for SEO and search: use the exact wording of the error (i think is as you put it in the FAQ but good to double check)